### PR TITLE
Add Bangla (bn) localization

### PR DESCRIPTION
**Pull request by eliyas5044**
This pull request includes a

- [ ] Bug fix
- [ ] New feature
- [x] Translation

Hello, i added new localization of Bangla language (bn).

### CODE REVIEW

**Comment by pedrofurtado** [comment]
@eliyas5044 Please, remove all files of PR from `dist`. These files are generated automatically in build.

File: src/js/select2/i18n/bn.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+define(function () {
+    // Bangla
```
Comment (pedrofurtado) [inline src/js/select2/i18n/bn.js]: This comment is not necessary.

**Comment by eliyas5044** [comment]
@pedrofurtado sorry, i am late, are you talking about to delete `dist` folder and send pull request again ?

**Comment by pedrofurtado** [comment]
@eliyas5044 The only file needed in your PR is the `src/js/select2/i18n/bn.js`. The other files in `dist` folder are generated automatically during build process of a new version :+1: 

I have a doubt: Why did you need to change the Gruntfile?

After this, the PR is ok to be merged :tada:

**Comment by eliyas5044** [comment]
@pedrofurtado done bro :)

**Comment by pedrofurtado** [comment]
@eliyas5044 Thanks! :handshake:

**Pull request merged**